src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.BIO.Custom.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.Win32.SafeHandles;
+using System.Diagnostics;
+
+internal static partial class Interop
+{
+    internal static class CustomBio
```

Comment:
The p/invoke methods here should be in `Interop.Crypto`.  The CustomBio class shouldn't be a direct child of `Interop` (a child of Interop.Crypto seems reasonable).

And instead of "Custom", can we name it?  "Custom" just feels like "My" or "Sample"... a documentation prefix that's absent any context or purpose.  "ManagedMemoryBio"?  Something else?